app/styles/components/_footer.scss

```diff
@@ -16,19 +16,42 @@ $icon-size: 25px;
   background-color: $Theme-color-blue;
 }
 
-.Footer-content {
-  @include font-properties(sans, desktop, content, large);
+.Footer-links {
+  @include font-properties(sans, desktop, content, small);
+  
+  display: flex;
+  flex-direction: column;
+}
+
+.Footer-link {
 
-  margin-top: $Theme-margin-small;
+  + .Footer-link {
+    margin-top: $Theme-margin-tiny;
```

Comment:
This should probably be better off extracted to a utility and removed from here.